h2o-r/tests/testdir_algos/glm/runit_GLM_negbinomial_dispersion.R

```diff
@@ -0,0 +1,259 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+
+FUZZ_COUNT <- 100
```

Comment:
This test now takes a lot of time trying random datasets. R fails to train on quite a lot of the random datasets so when R fails I skip the test to have something to compare.

Decreasing the `FUZZ_COUNT` speeds up the computation by reducing the number of generated datasets for each scenario.

This is nice for finding bugs but I plan to create some fixed datasets for jenkins running tests. Do you have any tips for selecting such datasets?